docs/libsemigroups.bib

```diff
@@ -344,11 +349,16 @@ @misc{Martin2011aa
 
 @article{Mitchell2021aa,
 	author = {J. D. Mitchell and M. Tsalakou},
-	date-added = {2021-03-09 15:23:03 +0000},
-	date-modified = {2021-03-09 15:24:14 +0000},
-	journal = {preprint},
 	title = {An explicit algorithm for normal forms in small overlap monoids},
-	year = 2021
+	journal = {CoRR},
+	volume = {abs/2105.12125},
+	year = {2021},
```

Comment:
This paper appeared already maybe cite the published version ?

Reply:
Does this seem reasonable:
```bib
@article{Mitchell2023aa,
  author = {Mitchell, James D. and Tsalakou, Maria},
  title = {An explicit algorithm for normal forms in small overlap monoids},
  journal = {Journal of Algebra},
  volume = {630},
  year = {2023},
  doi = {10.1016/j.jalgebra.2023.04.019},
  url = {https://doi.org/10.1016/j.jalgebra.2023.04.019},
  pages = {394--433},
  publisher = {Elsevier BV},
}
```